lib/fb/jwt/auth/version.rb

```diff
@@ -1,7 +1,7 @@
 module Fb
   module Jwt
     class Auth
-      VERSION = "0.10.0"
+      VERSION = "1.0.0"
```

Comment:
I don't mind changing this to `0.11.0` if we prefer that.
I did a major bump to show we are now on Ruby 3.